### PR TITLE
libgcc: Enable decimal float support for AArch64

### PR DESCRIPTION
**Pull request by Qwinci**
Enable decimal float support on AArch64 libgcc. This patch should probably be upstreamed to gcc at some point too.

### CODE REVIEW

**Comment by ArsenArsen** [comment]
we have a gcc fork at managarm/gcc that we use - please send a PR there (see comment in other PR, I didn't see this notification before commenting - my bad)

**Comment by Qwinci** [comment]
> we have a gcc fork at managarm/gcc that we use - please send a PR there (see comment in other PR, I didn't see this notification before commenting - my bad)

I opened a PR there and modified this one to just bump the revisions.

**Review by no92 — APPROVED** [review]
LGTM once managarm/gcc#1 is merged.

**Comment by Qwinci** [comment]
I guess this also needs a tag bump on the gcc repository and in this PR, Ill modify this one when the gcc PR gets merged and a new tag is created.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
LGTM but please drop the revisions from the gcc package and the gcc tools.

**Review by ArsenArsen — COMMENTED** [review]
missclick - sorry

lgtm, reset revision though

**Review by qookei — APPROVED** [review]

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**
